tests/src/JIT/Performance/CodeQuality/Inlining/InlineGCStruct.cs

```diff
@@ -0,0 +1,121 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.Xunit.Performance;
+using System;
+using System.Runtime.CompilerServices;
+using Xunit;
+
+[assembly: OptimizeForBenchmarks]
+[assembly: MeasureInstructionsRetired]
+
+public class InlineGCStruct
+{
+
+#if DEBUG
+    public const int Iterations = 1;
+#else
+    public const int Iterations = 2500000;
+#endif
+
+[MethodImpl(MethodImplOptions.NoInlining)]
+public static int FastFunctionNotCallingStringFormat(int param)
```

Comment:
It would be nice to have a comment that explains what the expected behavior is for this new test case.
I assume that you don't want the String.Format calls to be inlined here.
If someone have to investigate this as a failure, (like if we ran this test with JIT64 or JIT32) it would be good 
have an explanation about what we want to happen.
You also might want to reference this PR https://github.com/dotnet/coreclr/pull/7727 as well in the comment.